app/views/schemes/show.html.erb

```diff
@@ -14,24 +14,48 @@
     <%= render SubNavigationComponent.new(items: scheme_items(request.path, @scheme.id, "Locations")) %>
 
     <h2 class="govuk-visually-hidden">Scheme</h2>
-    <%= govuk_inset_text(text: edit_scheme_text(@scheme, current_user)) %>
-
-    <%= govuk_summary_list do |summary_list| %>
-      <% display_scheme_attributes(@scheme).each do |attr| %>
-        <%= summary_list.row do |row| %>
-          <% row.key { attr[:name] } %>
-          <% row.value do %>
-            <%= details_html(attr) %>
-            <% if attr[:name] == "Status" && @scheme.confirmed? && @scheme.locations.confirmed.none? && LocationPolicy.new(current_user, @scheme.locations.new).create? %>
-              <span class="app-!-colour-muted">Complete this scheme by adding a location using the <%= govuk_link_to("‘locations’ tab", scheme_locations_path(@scheme)) %>.</span>
-            <% end %>
-          <% end %>
-          <% if SchemePolicy.new(current_user, @scheme).update? %>
```

Comment:
is the scheme update policy still needed? or should it be changed at all in the light of this PR?

Reply:
I think at this point the policy is more around whether the user is someone who should be able to update the scheme (whether they're in the right org/role) as opposed to the state of the scheme, so I think it's still worth keeping around